CONTRIBUTING.md

```diff
@@ -95,3 +95,25 @@ Entrypoint of CICD is the file .gitlab-ci.yml.
 
 There you can find the full definition of the CI/CD pipeline.
 For more details see the [CICD wiki in our Dev Guide](https://github.com/vmware/versatile-data-kit/wiki/Gitlab-CICD)
+
+# How to make a new VDK release
+
+There are two kinds of releases made - development releases and stable releases.
+
+Development releases are made automatically and separately for each component when
+any changes to their directory are merged to the main branch. These have the `dev` tag
+in their version and are labelled `pre-release` in PyPI, meaning to install them by
+default one must include the `--pre` flag in their `pip install` call.
```

Comment:
I didn't see an usage of `--pre` flag in the MR, my impression from the documentation is, it would apply to all the dependencies recursively (direct and transient ones). I would suggest specifying a concrete pre-release version if needed in favour of `--pre` flag in that case

Reply:
Given dev releases are intended for development, it seems reasonable to pull dev builds for all packages. Unless you mean that dev builds will be pulled for external dependencies as well?